### ISSUE TITLE
pip install error -  No such file or directory: /flask-minify/requirements/main.txt

### ISSUE THREAD

**Issue by luaz** [mrf345/flask_minify#44]
I am using pyenv with Python 2.7

```bash
pip install Flask-Minify
```
```text
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
Collecting Flask-Minify
  Using cached Flask-Minify-0.27.tar.gz (6.8 kB)
    ERROR: Command errored out with exit status 1:
     command: /xxx/.pyenv/versions/2.7.18/envs/food2/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-CtdDDJ/flask-minify/setup.py'"'"'; __file__='"'"'/tmp/pip-install-CtdDDJ/flask-minify/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-Uffpej
         cwd: /tmp/pip-install-CtdDDJ/flask-minify/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-CtdDDJ/flask-minify/setup.py", line 18, in <module>
        with open(path.join(requirements_path, 'main.txt')) as f:
    IOError: [Errno 2] No such file or directory: '/tmp/pip-install-CtdDDJ/flask-minify/requirements/main.txt'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

I am not familiar with python package, but seems like `setup.py` try to access `requirements/*` which is not packaged in `Flask-Minify-0.28.tar.gz`. 

It works fine if I install from the source.

**Comment by mrf345** [comment]
Thanks for the well-detailed issue! I'm about to merge a PR that hopefully resolves it #45 . Please do let me know if the issue persists.

**Comment by mrf345** [comment]
`pip install Flask-Minify==0.29`
